test/org.hdfgroup.hdfview.test/uitest/AbstractWindowTest.java

```diff
@@ -371,6 +371,7 @@ protected void checkFileTree(SWTBotTree tree, String funcName, int expectedRowCo
             throw new IllegalArgumentException("SWTBotTree parameter is null");
         if (filename == null)
             throw new IllegalArgumentException("file name parameter is null");
+        bot.sleep(500);
```

Comment:
Seems fine but almost always a necessary sleep means something is wrong elsewhere

Reply:
I know - but it is in the SWTBOT/visual code (I think) because basically the event happens but before finishing the drawing/writing it tries to read before everything is complete. Adjusting the overall delay helped, but this was a critical check.